### ISSUE TITLE
 [ERROR] Migration failed. Check output above

### ISSUE THREAD

**Issue by cloudwales** [mplodowski/formbuilder-plugin-public#15]
I'm getting an issue when installing the plugin, where its telling me the migration has failed. The connection is fine and the DB works (been an active install for the last week and installed other plugins).

Heres the error::
```
❯ php artisan project:sync
Synchronizing Project...
All packages already installed
Executing: composer update
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Nothing to modify in lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating autoload files
> System\Console\ComposerScript::postAutoloadDump
Discovered Package: laravel/tinker
Discovered Package: nesbot/carbon
Package manifest generated successfully.
87 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> System\Console\ComposerScript::postUpdateCmd

* You are using October CMS version: v3.0.11
Checking Dependencies...
All dependencies installed
Checking Themes...
All themes checked
Executing: php artisan october:migrate

Migrating application and plugins...
System
Nothing to migrate.
Backend
Nothing to migrate.
Cms
Nothing to migrate.
Editor
Nothing to migrate.
Media
Nothing to migrate.
RainLab.Deploy
- Nothing to update.
Renatio.FormBuilder

 [ERROR] Migration failed. Check output above
```

October CMS Build: 3.0.11

Another odd thing is that the menu item is in the backend, but when you go into the Package manager, its not there, nor is it in the plugin folder.

Any ideas?

Many thanks

**Comment by mplodowski** [comment]
Hi,

this plugin is not yet tested with October v3 (beta).

I will look into this, but I cannot give you any estimation.

**Comment by cloudwales** [comment]
Ah ok, that makes sense. I'll take see look over today to see if i can help out. 

Cheers

**Comment by cloudwales** [comment]
Had a new message when playing with it today. Managed to get it working on Build 3.0.8.

Had to update all the migrations:

```
before:
use October\Rain\Support\Facades\Schema;

after:
use Schema;
```

This allowed me to migrate and all working on build 3.0.8 so far.

**Comment by mplodowski** [comment]
Fixed in version 2.1.7.
